src/Mono.Android/Android.Content.PM/PackageManager.cs

```diff
@@ -0,0 +1,79 @@
+namespace Android.Content.PM;
+
+#if ANDROID_34
+public abstract partial class PackageManager
+{
+	public sealed partial class PackageInfoFlags
+	{
+		// Create overloads that accept PackageInfoLongFlags
+		[global::System.Runtime.Versioning.SupportedOSPlatformAttribute ("android33.0")]
+		public static unsafe Android.Content.PM.PackageManager.PackageInfoFlags Of (PackageInfoLongFlags value)
```

Comment:
Why are `Of()` and `ValueAsFlags` are marked as `unsafe`?

Reply:
Copy/paste error, removed.